src/sentry/static/sentry/app/views/projectChooser.jsx

```diff
@@ -36,15 +36,15 @@ const ProjectChooser = createReactClass({
   redirectNoMultipleProjects() {
     let org = this.getOrganization();
     let projects = org.projects;
-    let task = TodoList.TASKS.filter(
+    let tasks = TodoList.TASKS.filter(
       task_inst => task_inst.task == this.props.location.query.task
-    )[0];
+    );
 
     if (projects.length === 0) {
       browserHistory.push(`/organizations/${org.slug}/projects/new/`);
-    } else if (projects.length === 1) {
+    } else if (projects.length === 1 && tasks.length === 1) {
```

Comment:
What if there is more than 1 task would we still want to redirect to the first, i.e.  `tasks.length >= 1`?

Reply:
The only way to do that is if the user types that into the URL, and in that situation, it feels like error'ing is the better (so we at least get notification).